german/finger-ins-loch/finger-ins-loch.md

```diff
@@ -5,3 +5,11 @@ Ihnen sticht eine Kiste ins Auge. Sie öffnen sie,
 finden darin einen Lappen, ziehen den Finger 
 aus dem Loch und stopfen den Lappen ins Loch. 
 Nun müssen Sie schnellstmöglichst ans Ufer.
+
+Dazu …
+
+… springen Sie [ins Wasser](ins-Wasser/ins-Wasser.md) und schieben das Boot an Land.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.